packages/mask/src/plugins/Avatar/Application/UploadAvatarDialog.tsx

```diff
@@ -10,10 +10,9 @@ import type { BindingProof } from '@masknet/shared-base'
 import { useI18N } from '../locales/i18n_generated'
 import { context } from '../context'
 import { useSubscription } from 'use-subscription'
-import type { NetworkPluginID } from '@masknet/web3-shared-base'
 import { useCurrentWeb3NetworkPluginID } from '@masknet/plugin-infra/web3'
 import { AvatarInfo, useSave } from '../hooks/save/useSave'
-import type { AllChainsNonFungibleToken } from '../types'
+import { SelectTokenInfo, SET_NFT_FLAG } from '../types'
```

Comment:
`SET_NFT_FLAG` is not clear enough. I see that it's used across multiple files, but I can't understand it by its name.